localstack/services/s3/v3/provider.py

```diff
@@ -309,8 +309,16 @@ def on_after_init(self):
         modify_service_response.append(S3Provider.service, s3_presigned_url_response_handler)
         register_website_hosting_routes(router=ROUTER)
 
+    def accept_state_visitor(self, visitor: StateVisitor):
+        visitor.visit(s3_stores)
+        visitor.visit(AssetDirectory(self._storage_backend.root_directory))
```

Comment:
note: this won't work for community cloud pods, as there's a check that the path should start with `config.dirs.data`

Reply:
acceptable limitation for now :+1: